xUnitCalculatorTests/xUnitCalculatorTests/CalculatorXUnitTests.cs

```diff
@@ -0,0 +1,92 @@
+using CSharpCalculator;
+
+namespace xUnitCalculatorTests
```

Comment:
Why does another HW is in PR for selenium Basics? You should create a branch from MAIN